### PR TITLE
Load tests

### PR DESCRIPTION
**Pull request by nstogner**
The beginnings of load tests

### CODE REVIEW

File: skaffold.yaml [hunk 1]
```diff
@@ -4,7 +4,7 @@ metadata:
   name: golang-proxy
 build:
   artifacts:
-    - image: lingo
+    - image: us-central1-docker.pkg.dev/substratus-dev/substratus/lingo
```
Comment (samos123) [inline skaffold.yaml:7]: Any reason why you use GAR here? We have lingo published for free to ghcr.

Comment (nstogner) [reply skaffold.yaml:7]: Not really, I wanted to be able to use skaffold from my local machine (I guess that doesnt matter here) and also didnt want to need to rely on other github actions running. Any thoughts?

Comment (samos123) [reply skaffold.yaml:7]: In that case you can revert the change to the image and it should all just work

Comment (samos123) [reply skaffold.yaml:7]: NVM I misunderstood how this worked outside of kind.

**Review by samos123 — APPROVED** [review]

**Pull request merged**